pandas/core/frame.py

```diff
@@ -174,6 +174,18 @@
 
     .. versionadded:: 0.17.0
 
+validate : {None, '1:1', '1:m', 'm:1', 'm:m', "one_to_one", "one_to_many",
+            "many_to_one", "many_to_many"}, default None
```

Comment:
Yes, I agree with the idea of not checking that a 'many' is in practice a 'one'. So this means that 'many_to_many' should check nothing.

But apart from that, we should either not have it as an option, or either include it in the docstring (and just say that this does do nothing)